test/Passes/tile-and-fuse-default.mlir

```diff
@@ -729,3 +729,58 @@ func.func @contraction(%arg0: tensor<16x1xf32>, %arg1: tensor<1x32xf32>) -> tens
 
 // CHECK-LABEL: contraction
 // CHECK-NOT: scf.for
+
+// -----
+
+// CHECK-LABEL: dlti_tile_size_32
+module attributes { dlti.target_system_spec = #dlti.target_system_spec<"CPU" : #dlti.target_device_spec<#dlti.dl_entry<"tile_size", 32 : i32>>> } {
```

Comment:
We usually want to pass more than one tile dim. How about `tile_size` gets replicated across all dims, and `tile_0`, `tle_1`, etc are specific? @adam-smnk

Reply:
Ideally, I agree we should have more fine-grained control. But it depends how far we want to push this integration right now.
I think it's good enough for now. For basic benchmarking with nice shapes it should be fine to set all tiles to 64, for example.

To do it really "properly" we'd have to define what convention we want to follow, what these tiles even mean etc.